airflow/providers/telegram/operators/telegram.py

```diff
@@ -0,0 +1,80 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Operator for Telegram"""
+from typing import Optional
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.telegram.hooks.telegram import TelegramHook
+from airflow.utils.decorators import apply_defaults
+
+
+class TelegramOperator(BaseOperator):
+    """
+    This operator allows you to post messages to Telegram using Telegram Bot API.
+    Takes both Telegram Bot API token directly or connection that has Telegram token in password field.
+    If both supplied, token parameter will be given precedence.
+
+    :param telegram_conn_id: Telegram connection ID which its password is Telegram API token
+    :type telegram_conn_id: str
+    :param token: Telegram API Token
+    :type token: str
+    :param chat_id: Telegram chat ID for a chat/channel/group
+    :type chat_id: str
+    :param text: Message to be sent on telegram
+    :type text: str
+    :param telegram_kwargs: Extra args to be passed to telegram client
+    :type telegram_kwargs: dict
+    """
+
+    template_fields = ('text', 'chat_id')
+    ui_color = '#FFBA40'
+
+    @apply_defaults
+    def __init__(
+        self,
```

Comment:
```suggestion
        self, *,
```
All args have to be a keywords.